pandas/core/frame.py

```diff
@@ -839,12 +839,12 @@ def itertuples(self, index=True, name="Pandas"):
             The name of the returned namedtuples or None to return regular
             tuples.
 
-        Yields
+        Returns
         -------
-        collections.namedtuple
-            Yields a namedtuple for each row in the DataFrame with the first
-            field possibly being the index and following fields being the
-            column values.
+        iterator
```

Comment:
What is this being changed for?

Reply:
Please see this [comment](https://github.com/pandas-dev/pandas/issues/25356#issuecomment-487398793).